streams-plugins/streams-plugin-pojo/src/main/java/org/apache/streams/plugins/StreamsPojoGenerationConfig.java

```diff
@@ -106,10 +106,10 @@ public boolean isIncludeJsr303Annotations() {
     return true;
   }
 
-  @Override
-  public boolean isUseCommonsLang3() {
-    return true;
-  }
+//  @Override
```

Comment:
1. What _was_ this function being used for? I don't see any references to it elsewhere in the repo.

2. Why comment-out instead of just removing the function entirely?

Reply:
older version of abstract parent class from jsonschema2pojo contained this method, but newer version does not.  it should be removed as you mentioned.